audio/decoders/wave.cpp

```diff
@@ -88,13 +89,29 @@ bool loadWAVFromStream(Common::SeekableReadStream &stream, int &size, int &rate,
 	uint16 blockAlign = stream.readUint16LE();	// == NumChannels * BitsPerSample/8
 	uint16 bitsPerSample = stream.readUint16LE();	// 8, 16 ...
 	// 8 bit data is unsigned, 16 bit data signed
-
+	fmtRemaining -= 16;
+
+	uint16 samplesPerBlock = 1;
+	// TODO: If type == kWaveFormatMSADPCM, there is a samplesPerBlock in the header
```

Comment:
Are there any games using that format? We should throw a warning in this case

Reply:
A grep of the codebase doesn't turn up anything (only other usage is in `video/avi_decoder.cpp`), but that doesn't guarantee anything. I've seen some places use the number instead of the constant (For example: the HE code uses `17` instead of the `kWaveFormatMSIMAADPCM` constant. Maybe I should change that with this PR too).

I will add a warning to this PR and repush.

Reply:
Done